app/controllers/users_controller.rb

```diff
@@ -66,6 +77,18 @@ def admin_user
     redirect_to root_url unless current_user.admin?
   end
 
+  def following
+    @title = "Following"
```

Comment:
1. chưa i18n
2. không nhét như này, nó sai rest rồi. Chô này chú sẽ phải nhét cả following và followers vào trong 1 action như show hay index của relationships_controller chả hạn. 1 action duy nhất nhé 
3. các method mà dùng gọi trên filter thì nhét vào trong private nhé, sửa lại toàn bộ file

Reply:
cái này khó quá bỏ qua hả chú ?

Reply:
Em fix rồi mà anh